subaru_global_2017_generated.dbc

```diff
@@ -241,6 +241,25 @@ BO_ 1677 Dash_State: 8 XXX
  SG_ COUNTER : 8|4@1+ (1,0) [0|15] "" XXX
  SG_ Units : 29|3@1+ (1,0) [0|7] "" XXX
 
+BO_ 1935 EYESIGHT_UDS_RESPONSE: 8 XXX
+ SG_ PCI : 0|8@1+ (1,0) [0|255] "Protocol Control Info" XXX
+ SG_ SID : 8|8@1+ (1,0) [0|255] "Service Identifier" XXX
+ SG_ DID : 16|16@1+ (1,0) [0|65535] "Data Identifier" XXX
+ SG_ Main : 32|1@1+ (1,0) [0|65535] "" XXX
+ SG_ Set : 33|1@1+ (1,0) [0|65535] "" XXX
+ SG_ Resume : 34|1@1+ (1,0) [0|65535] "" XXX
+ SG_ FCW : 32|1@1+ (1,0) [0|65535] "" XXX
+ SG_ AEB : 32|1@1+ (1,0) [0|65535] "" XXX
+
+SG_MUL_VAL_ 1935 DID SID 98-98;
+
+SG_MUL_VAL_ 1935 Main DID 243-243;
```

Comment:
One thing thats kinda weird about this is "243" is related to which DID we chose within OP to set as the "button" uds signal, so its kinda arbitrary.